common/src/main/res/layout/fragment_integration_portal.xml

```diff
@@ -14,385 +14,389 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<androidx.constraintlayout.widget.ConstraintLayout
+<androidx.swiperefreshlayout.widget.SwipeRefreshLayout
```

Comment:
This file wasn't actually changed as much as the diff seems to suggest - the entire layout was simply wrapped into `SwipeRefreshLayout`.